src/collector/linux-info-collector.js

```diff
@@ -0,0 +1,21 @@
+const exec = require('child_process').execSync;
+const Host = require('../model/host');
+
+function getHostOSVersion() {
+  return exec('cat /etc/redhat-release').toString();
```

Comment:
`exec` command be failed in some weird reason. 
Please attach TODO or something bcz we dont have time lol